cache.c

```diff
@@ -209,7 +209,7 @@ void cache_remove_ti(struct track_info *ti)
 static int read_cache(void)
 {
 	unsigned int size, offset = 0;
-	struct stat st = {};
+	struct stat st;
```

Comment:
Then that is a bug in the compiler you are using. Section 6.7.8 paragraph 19 says that

>all subobjects that are not initialized explicitly shall be initialized implicitly the same as
>objects that have static storage duration.